public/wearables/0000000000000000000000000000000000000000000000000000000000000170.json

```diff
@@ -1 +1 @@
-{"description":"# Description\n\n\n\n### Contributor\n\nSunflower Land is a community game built by a hundreds of developers and artists across the globe.\nCome join us on [Github](https://github.com/sunflower-land/sunflower-land)","decimals":0,"attributes":[{"trait_type":"Part","value":"Shirt"},{"trait_type":"Tradable","value":"Yes"}],"external_url":"https://bumpkins.io","image":"https://sunflower-land.com/play/wearables/images/estion_mark.png","name":"Ankh Shirt"}
\ No newline at end of file
+{"description":"# Description\n\n\n\n### Contributor\n\nSunflower Land is a community game built by a hundreds of developers and artists across the globe.\nCome join us on [Github](https://github.com/sunflower-land/sunflower-land)","decimals":0,"attributes":[{"trait_type":"Part","value":"Shirt"},{"trait_type":"Tradable","value":"Yes"}],"external_url":"https://bumpkins.io","image":"https://sunflower-land.com/play/wearables/images/8.png","name":"Ankh Shirt"}
```

Comment:
Strangely these appear to be pointing to the wrong image - `https://sunflower-land.com/play/wearables/images/8.png`

Should this be ID: `368` instead of `8`?